GeneratorInterface/Core/plugins/ExternalGeneratorFilter.cc

```diff
@@ -155,6 +155,11 @@ namespace externalgen {
     //Only stream 0 sets this at stream end Lumi and it is read at global end Lumi
```

Comment:
Is this comment still correct?

Reply:
Good question. The comment is definitely wrong and I will delete it. I'm still looking at this. I think I can make the data member that follows const or constexpr...

Reply:
I deleted the comment and changed randomState_ in the LumiCache from ```CMS_THREAD_SAFE mutable``` to ```const```.